<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPClient</class>
 <widget class="QMainWindow" name="TCPClient">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="theSendBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>121</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wyślij</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="theMessageTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="thePortTextEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="plainText">
     <string>3000</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="theHostTextEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="documentTitle">
     <string notr="true"/>
    </property>
    <property name="plainText">
     <string notr="true">localhost</string>
    </property>
   </widget>
   <widget class="QLabel" name="thePortLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="theHostLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="theConnectBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Połącz</string>
    </property>
   </widget>
   <widget class="QToolButton" name="theExplorerWnd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>51</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
